packages/api/src/external/carequality/document/get-non-existent-doc-refs.ts

```diff
@@ -47,18 +47,12 @@ async function checkDocRefsExistInS3(
     documents,
     async doc => {
       try {
-        const metriportId = doc.metriportId;
-
-        if (!metriportId) {
-          throw new Error(`Document ${doc.id} does not have a metriportId`);
-        }
-
-        const fileName = createFileName(cxId, patientId, metriportId);
+        const fileName = createFileName(cxId, patientId, doc.metriportId);
 
         const { exists } = await s3Utils.getFileInfoFromS3(fileName, s3BucketName);
 
         successfulDocs.push({
-          docId: metriportId,
+          docId: doc.metriportId,
```

Comment:
why change this? seems like we added this check for a reason?

Reply:
https://metriport.slack.com/archives/C04CXLSAF7V/p1709094435762369?thread_ts=1709075879.242879&cid=C04CXLSAF7V